### ISSUE TITLE
Servo tutorial frame errors

### ISSUE THREAD

**Issue by tylerjw** [ros-planning/moveit2_tutorials#172]
### Description

When you run the servo tutorial here: http://moveit2_tutorials.picknik.ai/doc/realtime_servo/realtime_servo_tutorial.html

The Fixed frame world does not seem to exist.  If I change this to panda_link0 then the PlanningScene plugin gives me the warning "Requesting initial scene failed" and the terminal is spammed with warnings from rviz2 that look like this:

```
[rviz2-1] Warning: Invalid frame ID "world" passed to canTransform argument source_frame - frame does not exist
[rviz2-1]          at line 133 in /tmp/binarydeb/ros-foxy-tf2-0.13.12/src/buffer_core.cpp
```

Does anyone know how I should troubleshoot this?  The keyboard control still works just fine.  I tested this on foxy with the debain install of moveit (as the tutorials instruct).

**Comment by sea-bass** [comment]
I don't think this issue pertains to the new version of MoveIt Servo, especially as the demos were also updated, so I think it's safe to close.
